frontend/server/src/main/java/org/pytorch/serve/job/Job.java

```diff
@@ -23,19 +23,14 @@ public Job(String modelName, String version, WorkerCommands cmd, RequestInput in
         begin = System.nanoTime();
         scheduled = begin;
 
-        Map<String, String> headers = input.getHeaders();
-        if (headers.containsKey("X-TS-Priority")) {
-            this.priority = Integer.parseInt(headers.get("X-TS-Priority"));
-        } else {
-            this.priority = 0;
-        }
+        this.priority = input.getHeaders().getOrDefault("x-ts-priority", "max");
```

Comment:
I think we discussed this before, but I thought we discusses about python code -> I am not sure if we will get headers always as lower case or upper case -> maybe we should indeed go over all keys and check if they match the `x-ts-priority` header case insensitive.

Reply:
Yes, I agree! On branch `torchserve-23mt`, I am just [lowercasing all headers](https://github.com/pytorch/serve/commit/88e6e924e9d9659274a1ecf3dbf1c096b6a7aa36#diff-4290dace1516935aa9d107cf7a46b7c55114a4229bca4cd69756286b08657f1f) upon receiving (`Java` does not seem do this out of the box). I can apply this fix here as well, since it seems important to not silently use the wrong priority.